clab/config.go

```diff
@@ -433,6 +436,35 @@ func (c *CLab) verifyLinks() error {
 	return nil
 }
 
+// verifyDuplicateAddresses checks that every static IP address in the topology is unique
+func (c *CLab) verifyDuplicateAddresses() error {
+	var ipv4_list []string
+	var ipv6_list []string
+	for _, node := range c.Nodes {
+		ipv4_addr := node.Config().MgmtIPv4Address
+		if ipv4_addr != "" {
```

Comment:
better to keep the happy path on the main line
meaning that 
```
if ipv4_addr == ""
  continue
```
is a better alternative